### PR TITLE
modify the drivation of multisig, fix minor bug at utils.go.

### PR DESCRIPTION
**Pull request by yenkhoon**

### CODE REVIEW

**Comment by yenkhoon** [comment]
@b00f if there is nothing else can merge the pull request?

File: app/app_test.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	"github.com/maxonrow/maxonrow-go/x/auth"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDerive(t *testing.T) {
+
+	delAddr1, _ := sdkTypes.AccAddressFromBech32("mxw1dk6d4g8gxcy3terzwfffn7cx5hn6sukg8xu6np")
+
+	derivedAddr := auth.DeriveMultiSigAddress(delAddr1, 0x987654321)
+	fmt.Println(derivedAddr.String())
```
Comment (b00f) [inline app/app_test.go]: Remove this line and try to add test cases for this address

File: app/app_test.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+	derivedAddr2 := auth.DeriveMultiSigAddress(delAddr1, 0x98765432100)
+	fmt.Println(derivedAddr2.String())
```
Comment (b00f) [inline app/app_test.go]: same here

Comment (yenkhoon) [reply app/app_test.go]: what thing same?

**Pull request merged**